content/tr/security-chaos-engineering.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Kaos Güvenliği Mühendisliği
+status: Completed
+category: concept
+tags: ["güvenlik", "metodoloji", ""]
+---
+
+Kaos güvenliği mühendisliği (SCE), [kaos mühendisliği](/chaos-engineering/) tabanlı bir disiplindir. SCE, sistemin çalkantılı ve kötü niyetli koşullara dayanma becerisine güven oluşturmak için dağıtılmış bir sistem üzerinde proaktif güvenlik deneyleri gerçekleştirir. Kaos güvenliği mühendisleri, bunu başarmak için kararlı durum, hipotez, sürekli doğrulama, öğrenilen dersler ve hafifletme uygulamasını kapsayan bilimsel yöntem döngülerini kullanır.
+
+## Hangi Sorunları Çözer
+[Site güvenilirliği mühendisleri](/site-reliability-engineering/) (SRE'ler) ve siber güvenlik mühendisleri için ana öncelik, sıfır kesintiye ulaşmak ve iş etkisini en aza indirme hedefiyle hizmeti olabildiğince hızlı bir şekilde geri yüklemektir. SRE'ler ve siber güvenlik mühendisleri hem arıza öncesi hem de arıza sonrası olay durumlarıyla ilgilenir. Çoğu güvenlik sorununun hızlı bir şekilde tespit edilip düzeltilmesi zordur, bu da uygulama veya sistem işlevselliğini etkiler. Ek olarak, güvenlik olaylarının geliştirme aşamasında ortaya çıkarılması genellikle zordur.
+
+## Nasıl Yardımcı Olur
+Kaos güvenliği mühendisliği, [gözlemlenebilirlik](/observability/) ve siber dayanıklılık uygulamaları üzerine kurulmuştur. "Bilinmeyen bilinmeyenleri" ortaya çıkarmayı ve sistemde güven oluşturmayı, siber dayanıklılığı artırmayı ve gözlemlenebilirliği iyileştirmeyi amaçlamaktadır. 
+Mühendislik ekiplerinin, karmaşık altyapı, platformlar ve dağıtılmış sistemlerdeki güvenlik sorunlarına ilişkin anlayışı giderek gelişecektir. SCE, tüm ürünün siber dayanıklılığını artırır, gizli güvenlik sorunlarını ortaya çıkarır, klasik kör noktaları ortaya çıkarır ve ekipleri kritik uç durumlara hazırlar. Bu yaklaşım, [SRE](/site-reliability-engineering/)'lerin, [DevOps](/devops/)'un ve [DevSecOps](/devsecops/) mühendislerinin sistemde güven oluşturmasına, siber dayanıklılığı artırmasına ve gözlemlenebilirliği iyileştirmesine yardımcı olur.
```

Comment:
```suggestion
Mühendislik ekiplerinin, karmaşık altyapı, platformlar ve dağıtılmış sistemlerdeki güvenlik sorunlarına ilişkin anlayışı giderek gelişecektir. SCE, tüm ürünün siber dayanıklılığını artırır, gizli güvenlik sorunlarını ortaya çıkarır, klasik kör noktaları ortaya çıkarır ve ekipleri kritik uç durumlara hazırlar. Bu yaklaşım, [SRE](./site-reliability-engineering/)'lerin, [DevOps](./devops/)'un ve DevSecOps mühendislerinin sistemde güven oluşturmasına, siber dayanıklılığı artırmasına ve gözlemlenebilirliği iyileştirmesine yardımcı olur.
```

Similar to above, some pages don't exist and some links are to the English ones.